### PR TITLE
Update write-i18n-loaders.js to return promises appropriately.

### LINKED ISSUE #10: Be more explicit about what's happening in the require.ensure block
I'm seeing links out to this warning (http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-was-not-returned-from-it) in a project that is enabling blue bird warnings.  I think we need to return null in our require.ensure.

### PR DESCRIPTION
**Pull request by ryanthemanuel**
Update write-i18n-loaders.js to return promises appropriately.  Not returning anything is an anti-pattern (http://bluebirdjs.com/docs/warning-explanations.html#warning-a-promise-was-created-in-a-handler-but-was-not-returned-from-it) Fixes #10

### CODE REVIEW

File: lib/write-i18n-loaders.js [hunk 1]
```diff
@@ -28,7 +28,10 @@ const createTranslationLoader = (loaderName, locale, format) => {
     return (
       `const ${loaderName} = (callback, scope) =>
    import( /* webpackChunkName: "${locale}-translations" */ '${locale}.js')
-     .then((module) => { callback.call(scope, module);})
+     .then((module) => { 
+       callback.call(scope, module);
+       return Promise.resolve(module);
```
Comment (emilyrohrbough) [inline lib/write-i18n-loaders.js]: do we want to return the promised value since it wasn't before?

**Review by mjhenkes — APPROVED** [review]

File: lib/write-i18n-loaders.js (hunk 1 above)
Comment (ryanthemanuel) [reply lib/write-i18n-loaders.js]: Good point

**Review by bjankord — APPROVED** [review]
Looks like there are some jest snapshots that need updated with this change.

**Review by emilyrohrbough — APPROVED** [review]

**Comment by emilyrohrbough** [comment]
Whats the status on this PR?

**Pull request merged**